app/assets/stylesheets/style.scss

```diff
@@ -1 +1,13 @@
+body {
+   display: flex;
+   min-height: 100vh;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered display, flex-direction, min-height